scilpy/reconst/dti.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+
+supported_tensor_formats = ['fsl', 'nifti', 'mrtrix', 'dipy']
+tensor_format_description = \
+    """
+    Dipy's order is [Dxx, Dxy, Dyy, Dxz, Dyz, Dzz]
+       Shape: [i, j , k, 6].
+       Ref: [See _lt_rows and _lt_cols in dipy.reconst.dti]
+    
+    MRTRIX's order is : [D11, D22, D33, D12, D13, D23]
```

Comment:
Add [D11, D22...] to the other formats or remove to have an "homogeneous" format presentation.

Reply:
The reason why the formatting is different is that I was copying the wording from the reference link to show I didn't make a mistake. But I can just remove the D11, D22 version. Dxx is already clear.